server/cloudinary.js

```diff
@@ -0,0 +1,22 @@
+const cloudinary = require('cloudinary');
+const config = require('../cloudinary.config.js');
+
+const uploadPhotoToCloudinary = (body) => {
+  cloudinary.config({
```

Comment:
Haven't finished my first cup of coffee yet, but I'm wondering if this config step (lines 5-9) should happen inside this function? I'm thinking it could happen outside the function, that way this "config" operation doesn't have to happen every time this function is invoked, since the values we're getting from the config file (cloud name, api key & secret) are pretty much constant right?

If so, this would simplify the function nicely!

Reply:
Hi @jabrake, thank you for your input! For some reason I thought that cloudinary.config was some sort of functions that must be invoked every time the API call was made. I just refactored and it seems to be working. I've already pushed the new version, great catch!